charts/partials/templates/_proxy-init.tpl

```diff
@@ -0,0 +1,40 @@
+{{- define "partials.proxy-init" -}}
+- args:
+  - --incoming-proxy-port
+  - {{.Proxy.Ports.Inbound | quote}}
+  - --outgoing-proxy-port
+  - {{.Proxy.Ports.Outbound | quote}}
+  - --proxy-uid
+  - {{.Proxy.UID | quote}}
+  - --inbound-ports-to-ignore
+  - {{.Proxy.Ports.Control}},{{.Proxy.Ports.Admin}}{{ternary (printf ",%s" .ProxyInit.IgnoreInboundPorts) "" (not (empty .ProxyInit.IgnoreInboundPorts))}}
+  - --outbound-ports-to-ignore
+  - {{.ProxyInit.IgnoreOutboundPorts | quote}}
+  image: {{.ProxyInit.Image.Name}}:{{.ProxyInit.Image.Version}}
+  imagePullPolicy: {{.ProxyInit.Image.PullPolicy}}
+  name: linkerd-init
+  {{- include "partials.resources" .ProxyInit.Resources | nindent 2 }}
```

Comment:
Looks like we do need the line feed when including resources, so it should be `{{` instead of `{{-`

Reply:
do not*

Reply:
On my end, it looks right:
```
        name: linkerd-init
        resources:
          limits:
            cpu: 100m
            memory: 50Mi
          requests:
            cpu: 10m
            memory: 10Mi
```
What does it look like on your end?

Reply:
```
name: linkerd-initresources:                               
  limits:                                               
    cpu: 100m                                                 
    memory: 50Mi           
  requests:                                              
    cpu: 10m                                                      
    memory: 10Mi 
```
The difference in my setup is that I've modified the partials by removing the initial dash, so all the properties are at the root of the document, and apparently `{{-` eats the preceding line feed when it's placed at the beginning of the line...

Reply:
Per _slack_ convo, we will change the content of the proxy, proxy-init, resources and debug partials to YAML objects, instead of an element in a YAML list, to ensure the `toJson` function on the `inject` side can convert it to a JSON object (as needed by the JSON patch), instead of a JSON list.